### PR TITLE
feat: navbar and checkbox, radio, search input types

### LINKED ISSUE #12: Navigation component
### Navigation component
- [x] Simple navigation

### PR DESCRIPTION
**Pull request by siddhigate**
### Input types
- Checkbox
- Radio
- Search

### Navigation
- Simple navbar

### Live Preview
**Checkbox Radio:** https://upsleek-css-component-library-oqjgjxgfx-siddhigate.vercel.app/docs/form/#checkbox-radio
**Search input:** https://upsleek-css-component-library-oqjgjxgfx-siddhigate.vercel.app/docs/form/#search-input
**Simple navbar:** https://upsleek-css-component-library-oqjgjxgfx-siddhigate.vercel.app/docs/nav/

### Snapshots
image
image

image

### CODE REVIEW

File: css/components/inputs/radio.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+.radio-container .radio-custom {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 20px;
+  height: 20px;
+  background-color: var(--clr-secondary-light);
+  border-radius: 50%;
+  transition: 0.3s background-color;
+}
```
Comment (msdeshmukh009) [inline css/components/inputs/radio.css]: Consider keeping radio and checkbox styles together as there are multiple lines of code that are similar in both.

File: css/components/nav.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.header .search-input-wrapper {
+  flex-grow: 1;
+}
+
+.navigation {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (msdeshmukh009) [inline css/components/nav.css]: Maybe make utility class for 
```
 display: flex;
  align-items: center;
```

**Review by msdeshmukh009 — APPROVED** [review]
Looks good to me.

**Review by manojasarsa — APPROVED** [review]
Looks good!

File: css/components/inputs/radio.css (hunk 1 above)
Comment (siddhigate) [reply css/components/inputs/radio.css]: Thanks for your review! Code is optimized now.

**Review by ZuberDunge — APPROVED** [review]
Excellent code structure, UI looks pretty good. Everything looks fine to me. Good to go from my side. Good work Siddhi.

**Comment by preethasri** [comment]
UI looks good 👍

**Pull request merged**